.eslintrc.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     "prefer-rest-params": "off",
     "react/require-extension" : "off",
     "import/no-extraneous-dependencies" : "off",
-    "no-console": "off"
+    "no-console": "off",
+    "no-param-reassign": "off",
```

Comment:
please, don't disable linter rules for the whole project. If there is a case in which you need to disable this rule, do it inline (but I think it's not necessary)